execution_engine_testing/tests/benches/auction_bench.rs

```diff
@@ -0,0 +1,297 @@
+use std::{path::Path, time::Duration};
+
+use criterion::{
+    criterion_group, criterion_main, measurement::WallTime, BenchmarkGroup, Criterion, Throughput,
+};
+use rand::Rng;
+use tempfile::TempDir;
+
+use casper_engine_test_support::{
+    DeployItemBuilder, ExecuteRequestBuilder, LmdbWasmTestBuilder, StepRequestBuilder,
+    DEFAULT_ACCOUNT_ADDR, DEFAULT_ACCOUNT_INITIAL_BALANCE, DEFAULT_ACCOUNT_PUBLIC_KEY,
+    DEFAULT_AUCTION_DELAY, DEFAULT_GENESIS_CONFIG_HASH, DEFAULT_GENESIS_TIMESTAMP_MILLIS,
+    DEFAULT_LOCKED_FUNDS_PERIOD_MILLIS, DEFAULT_PROPOSER_PUBLIC_KEY, DEFAULT_PROTOCOL_VERSION,
+    DEFAULT_ROUND_SEIGNIORAGE_RATE, DEFAULT_SYSTEM_CONFIG, DEFAULT_UNBONDING_DELAY,
+    DEFAULT_WASM_CONFIG, MINIMUM_ACCOUNT_CREATION_BALANCE, SYSTEM_ADDR,
+};
+use casper_execution_engine::{
+    core::engine_state::{
+        genesis::GenesisValidator, run_genesis_request::RunGenesisRequest, EngineConfig,
+        ExecConfig, GenesisAccount, RewardItem,
+    },
+    shared::system_config::auction_costs::DEFAULT_DELEGATE_COST,
+};
+use casper_types::{
+    account::AccountHash,
+    runtime_args,
+    system::auction::{self, Bids},
+    Motes, ProtocolVersion, PublicKey, RuntimeArgs, SecretKey, U512,
+};
+
+const ARG_AMOUNT: &str = "amount";
+const ARG_TARGET: &str = "target";
```

Comment:
I think we already have these as public constants somewhere, might as well import them here instead of redefining.

Reply:
Only ARG_AMOUNT is provided by casper-test-support.